rails-commenting.rb

```diff
@@ -36,15 +36,15 @@ def delete
     if @post.destroy
       redirect_to blog_posts_path
     else
-      # 6)
+      # 6) If it fails to destroy the blog post, it will redirect you to the post you intended to delete
       redirect_to blog_post_path(@post)
     end
   end
 
-  # 7)
+  # 7) makes 
   private
   def blog_post_params
-    # 8)
+    # 8) something, suggestion made by connor 
```

Comment:
hmm missed a couple answers here (num 7 - 10), but for the most part great!

Reply:
.... oh man, I did not intend on uploading this! It was a joke I made with connor in class, I thought I had edited it! .... sorry.